DeliBuddy/app/src/main/java/yapp/android1/delibuddy/util/permission/CheckPermission.kt

```diff
@@ -0,0 +1,60 @@
+package yapp.android1.delibuddy.util.permission
+
+import android.content.pm.PackageManager
+import androidx.activity.result.ActivityResultLauncher
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.app.ActivityCompat.shouldShowRequestPermissionRationale
+import androidx.core.content.ContextCompat
+import yapp.android1.delibuddy.ui.dialog.PermissionDialogFragment
+
+object CheckPermission {
```

Comment:
```suggestion
object PermissionHelper {
```
PermissionHelper , PermissionManager 정도는 어떨까요?

Reply:
이름 변경하는 것이 훨씬 좋을 것 같아요! 변경해 보겠습니당~